android-base/src/main/java/io/github/droidkaigi/confsched2020/ui/widget/SystemUiManager.kt

```diff
@@ -58,8 +58,11 @@ class SystemUiManager(
             0
         } else {
             View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
+        } or if (isLandscape()) {
```

Comment:
@fornewid 
Thank you for your PR.

Changing the flags will affect the entire application, so I would like to achieve the same behavior using the insetter if possible, like so
https://github.com/DroidKaigi/conference-app-2020/pull/72/files#diff-d0a86a9871834b5287736b83beab9d37R79
Could you consider this?

Reply:
Thanks, I'll reconsider using insetter ASAP. :)

Reply:
Take as much time as you need :D
Thank you for your cooperation!